p2p/src/authenticated/actors/router/ingress.rs

```diff
@@ -50,22 +56,27 @@ impl<P: PublicKey> Mailbox<P> {
         receiver.await.unwrap()
     }
 
+    /// Notify the router that a peer is no longer available.
     pub async fn release(&mut self, peer: P) {
         self.sender.send(Message::Release { peer }).await.unwrap();
     }
 }
 
 #[derive(Clone, Debug)]
+/// Sends messages containing content to the router to send to peers.
 pub struct Messenger<P: PublicKey> {
     sender: mpsc::Sender<Message<P>>,
 }
 
 impl<P: PublicKey> Messenger<P> {
+    /// Returns a new [Messenger] with the given sender.
+    /// (The router has the corresponding receiver.)
     pub fn new(sender: mpsc::Sender<Message<P>>) -> Self {
         Self { sender }
     }
 
-    pub async fn content(
+    /// Sends a message to the given `recipients`.
+    pub async fn send(
```

Comment:
IIRC I opted to call this `content` to mirror the names of the functions on `Mailbox` (where they described the message they were sending).

For now, I'd recommend sticking with that approach just for the sake of consistency (as the other functions are called by different actors and they are all technically "sending").